test/images/Makefile

```diff
@@ -12,11 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+REPO_ROOT:=${CURDIR}/../..
 REGISTRY ?= registry.k8s.io/e2e-test-images
 GOARM ?= 7
 DOCKER_CERT_BASE_PATH ?=
 QEMUVERSION=v5.1.0-2
-GOLANG_VERSION=1.24.0
+GOLANG_VERSION=$(shell cat $(REPO_ROOT)/.go-version)
```

Comment:
/hold
we'll need to make the test/image builds trigger on .go-version if they weren't already

Reply:
actually, they don't trigger on changes to the makefile currently anyhow so ... this wouldn't be a regression, but we should fix that anyhow
/hold cancel

Reply:
https://github.com/kubernetes/test-infra/pull/34700